showcase.json

```diff
@@ -2998,8 +2998,26 @@
     "youtube": "https://www.youtube.com/@omnianetwork2947",
     "submittableId": "43839848"
   },
-
-  {
+    {
+    "name": "Flower Power DAO",
+    "description": "An NFT Trilogy and community-curated garden inspired by the French Artist, Ludo.",
+    "website": "https://fpdao.app",
+    "logo": "/img/showcase/fpdao_logo.webp",
```

Comment:
Hello @evanmcfarland, I believe that you uploaded PNGs. Can you please update it with the correct file type?